clients/graphql-kotlin-spring-client/build.gradle.kts

```diff
@@ -10,6 +10,7 @@ dependencies {
     api(projects.graphqlKotlinClientJackson)
     api(libs.kotlinx.coroutines.reactor)
     api(libs.spring.webflux)
+    api(libs.spring.context)
```

Comment:
why do we need explicit spring context dependency here?

Reply:
Spring changed their dependency management I guess 🤷

`:graphql-kotlin-spring-client:compileTestKotlin`

> Supertypes of the following classes cannot be resolved. Please make sure you have the required dependencies in the classpath:
>     class org.springframework.http.client.reactive.ReactorClientHttpConnector, unresolved supertypes: org.springframework.context.SmartLifecycle
>